### PR TITLE
Add String trim examples

### PR DESCRIPTION
**Pull request by Elchi3**
I am a little puzzled why there are no String examples yet... Did we decide against doing live examples for String or is this an oversight? 

Anyway, here is my first JS examples PR that creates examples for:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimStart
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trimEnd

### CODE REVIEW

File: live-examples/js-examples/string/string-trim.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var greeting = "   Hello world!   ";
```
Comment (ExE-Boss) [inline live-examples/js-examples/string/string-trim.html]: Shouldn’t we be using `let` or `const` here?

File: live-examples/js-examples/string/string-trimend.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var greeting = "   Hello world!   ";
```
Comment (ExE-Boss) [inline live-examples/js-examples/string/string-trimend.html]: ditto

File: live-examples/js-examples/string/string-trimstart.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var greeting = "   Hello world!   ";
```
Comment (ExE-Boss) [inline live-examples/js-examples/string/string-trimstart.html]: ditto

**Comment by wbamberg** [comment]
It's an oversight on my account :(. Thanks for the examples @Elchi3 , I will take a look when I get a minute.

File: live-examples/js-examples/string/string-trim.html (hunk 1 above)
Comment (wbamberg) [reply live-examples/js-examples/string/string-trim.html]: Thanks for the review @ExE-Boss , but the style guide says no: https://github.com/mdn/interactive-examples/blob/master/JS-Example-Style-Guide.md#language-choice-es5es6

**Pull request merged**